### PR TITLE
Fixes #6460 and #3214

### LINKED ISSUE #3214: Your species really, really, REALLY cannot wear that
Attempting to quickswap (ctrl+e) a piece of garment that's not suitable for your species will result in the error message being repeated quite a few times.
ss 2015-02-24 at 08 47 57

### LINKED ISSUE #6460: Hivebot meat
I was under the impression they were robots.  When killed (gibbed more accurately) they drop hivebot meat.

### PR DESCRIPTION
**Pull request by unid15**
- Fixes #3214 
- Fixes #6460 :
- It's no longer possible to obtain bee, juggernaut, artificer, wraith, slime, spiderbot, skeleton, drone, shade, viscerator, mining drone and puddi meat by gibbing those monsters (fun removal :sunglasses:). Russian, syndicate and wizard meat still exists though
- Mushrooms and viscerator are now small ( = can be picked up)

### CODE REVIEW

**Comment by FudgePucker** [comment]
Make puddi meat into its own thing with its own sprite.

File: code/modules/mob/living/simple_animal/constructs.dm [hunk 1]
```diff
@@ -33,6 +33,8 @@
 	mob_swap_flags = HUMAN|SIMPLE_ANIMAL|SLIME|MONKEY
 	mob_push_flags = ALLMOBS
 
+	meat_type = 0
```
Comment (DeityLink) [inline code/modules/mob/living/simple_animal/constructs.dm]: replace with "meat_type = /obj/item/weapon/ectoplasm"

File: code/modules/mob/living/simple_animal/shade.dm [hunk 2]
```diff
@@ -28,6 +28,7 @@
 	status_flags = CANPUSH
 	supernatural = 1
 	flying = 1
+	meat_type = null
```
Comment (DeityLink) [inline code/modules/mob/living/simple_animal/shade.dm]: replace with "meat_type = /obj/item/weapon/ectoplasm"

File: code/modules/mob/living/simple_animal/constructs.dm (hunk 1 above)
Comment (PJB3005) [inline code/modules/mob/living/simple_animal/constructs.dm]: What if the code assumes `meat_type` is a type of meat, and runtimes as a result of this?

Comment (DeityLink) [inline code/modules/mob/living/simple_animal/constructs.dm]: The code never checks if meat_type is an actual type of meat. I've tested it and it works fine.

**Pull request merged**